index.html

```diff
@@ -134,6 +141,17 @@ <h2 id="scope"> Scope </h2>
           Synchronization of web content among multiple connections may be
           possible, but is not defined by the specifications in this group.
         </p>
+        <p> The suite of network protocols will cover required steps for two
+          user agents to implement the APIs: discovery, authentication,
+          transport, and messaging (including API control messages and messages
+          for streaming media between agents). The suite will reuse and
+          configure existing IETF protocols for each of these steps. This
+          Working Group will work with IETF to assess whether parts of the
+          suite of network protocols that may apply to broader scenarios
+          than the ones envisioned for the APIs may be developed in IETF. </p>
```

Comment:
I would say:

> envisioned for the APIs.  If that is the case, those parts will be further developed in the IETF.

Reply:
I dropped the "that" to split the sentence into two, and integrated the suggested wording.